src/PreferencesDialog.ui

```diff
@@ -20,7 +20,7 @@
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
-      <number>0</number>
+      <number>4</number>
```

Comment:
The change in height (a few lines up) makes sense.  But this change of the `tabWidget.currentIndex` value doesn't seem right? :wink:

Reply:
I actually didn't pay much attention to the UI changes because I did it with Qt Creator, I'll fix that and test it.

Reply:
I'm not sure why it was changed, but it's fine to change it. I fixed this review too :)